keyboards/ymdk/ymd62/mcuconf.h

```diff
@@ -0,0 +1,31 @@
+/* Copyright 2020 QMK
+ * Copyright 2024 Richard Dawe (@richdawe)
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+/*
+ * This file was auto-generated by:
+ *    `qmk chibios-confmigrate -i keyboards/handwired/onekey/bluepill/mcuconf.h -r platforms/chibios/STM32_F103_STM32DUINO/configs/mcuconf.h`
+ */
+
+#pragma once
+
+#include_next <mcuconf.h>
+
+#undef STM32_PWM_USE_TIM4
+#define STM32_PWM_USE_TIM4 TRUE
+
+#undef STM32_SPI_USE_SPI2
+#define STM32_SPI_USE_SPI2 FALSE
```

Comment:
```suggestion
```

Reply:
This seems to be enabled by default for this microcontroller, see [its mcuconf.h](https://github.com/richdawe/qmk_firmware/blob/master/platforms/chibios/boards/STM32_F103_STM32DUINO/configs/mcuconf.h#L171).

Things seem to be work fine the defaults for SPI2, so I will remove this. Thanks!